framework/core/Project/Mockito.pm

```diff
@@ -97,6 +97,9 @@ sub _post_checkout {
     if (-e "$work_dir/gradle.properties") {
         system("sed -i.bak s/org.gradle.daemon=true/org.gradle.daemon=false/g \"$work_dir/gradle.properties\"");
     }
+
+    # Enable local repository
+    system("find $work_dir -type f -name \"build.gradle\" -exec sed -i.bak 's|jcenter()|maven { url \"$SCRIPT_DIR/lib/build_systems/gradle/gradle_deps\" }\\n jcenter()\\n|g' {} \\;");
```

Comment:
This essentially sets the URL for each dependency to a local path -- right?

Reply:
Kinda. This tells Gradle to first try to find all dependencies on a local repository before querying jcenter (the official gradle repository I suppose).